docs/docs/hpc-on-k8s/rke.md

```diff
@@ -0,0 +1,89 @@
+# Installing Kubernetes Cluster with RKE
+
+We are using RKE (Rancher Kubernetes Engine) to install a Kubernetes cluster.
+
+## Types of Nodes in Kubernetes Installations
+
+- **rke node (devops node):** This is the node where the RKE utility is installed, responsible for the installation of the Kubernetes cluster on other nodes.
+- **k8s nodes (master/worker nodes):** These are the nodes that are part of the Kubernetes cluster where the installation takes place.
+
+## Prerequisites (Master/Worker)
+
+- Docker must be installed on all nodes of the Kubernetes cluster.
+- Password-less SSH access must be set up from the rke node to all Kubernetes nodes.
+
```

Comment:
Experienced issues with bringing up `rke` before having firewall disabled (or appropriate rules setup for internal network communication) which were only fixed after making firewall correct and then restarting docker. 

It seems sensible then to make one of the prerequisites be firewall related

Reply:
updated thanks